src/main/kotlin/com/arsiu/eduhub/dto/response/LessonDtoResponse.kt

```diff
@@ -0,0 +1,8 @@
+package com.arsiu.eduhub.dto.response
+
+data class LessonDtoResponse(
+    val id: Long,
+    val name: String,
+    val chapterId: Long,
+    val assignments: List<AssignmentDtoResponse>?
```

Comment:
nullable list? Are u sure?

If u have strong reasons why it should be nullable then ok. Otherwise make it non nullable